src/Social/Contract/Interactions/InteractionsTrait.php

```diff
@@ -19,9 +19,9 @@ trait InteractionsTrait
      * @param string $action
      * @return void
      */
-    public function interact(string $action): void
+    public function interact(string $action, $data): void
```

Comment:
@arfenis can we know what data type $data is?

Reply:
Ups, forgot.